### PR TITLE
#19 Yaml created for metabase initialization

### PR DESCRIPTION
**Pull request by francisco1code**
## Motivation

Create a way to visualize the data in a simple and fast way, for this the metabase was used, .yml file was generated to create containers to run the metabase

## Warning

Make sure docker and docker-compose are installed
when running the first time create the network  ```docker network create metabase```

## Changes

-  added metabase

## Status Checklist

- [x] Test

## Screenshots

|Output|
|---|
||

## Testing

- Execute ```docker-compose -f metabase/metabase.yml up ```

### CODE REVIEW

**Comment by RochaCarla** [comment]
Add the instructions in the readme file. 

- Explain what the goal of metabase in the project
- Explain the commands necessary to execute
- How to test (configure a new database, for example)

Make sure docker and docker-compose are installed
when running the first time create the network  docker network create metabase

File: makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+clean:
+	docker-compose -f ${dkc} kill
+	docker-compose -f ${dkc} stop
+	docker-compose -f ${dkc} down --rmi local --volumes
+	docker-compose -f ${dkc} rm -f
+	@echo "Docker containers have been stopped and deleted."
+
+migrate:
+	@echo "Loading data into the database..."
+	docker cp data/data_example.csv mongo:/
```
Comment (LeoSilvaGomes) [inline makefile:18]: Probably we will need to add explanation about this line! What do you think to create a `make command` where the user can pass the file path?

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
-# Playground
\ No newline at end of file
+# Playground
+
+
```
Comment (LeoSilvaGomes) [inline README.md:3]: 🔪

Comment (LeoSilvaGomes) [reply README.md:3]: What do you think to add a little explanation about the project here! For more instructions talk to @alvesisaque

**Review by LeoSilvaGomes — APPROVED** [review]
Well done ✅

File: makefile (hunk 1 above)
Comment (francisco1code) [reply makefile:18]: added parameter in make migrate command

**Review by LeoSilvaGomes — APPROVED** [review]
you 🪨

**Comment by LeoSilvaGomes** [comment]
Please, never forget to add all the details in the pr, it is important for us to know how it is going in our kaban
image

**Comment by francisco1code** [comment]
Conflict resolved and merged

**Pull request merged**